### ISSUE TITLE
New rule: require class attributes to go after docstring and before all method definitions

### ISSUE THREAD

**Issue by DeflateAwning** [astral-sh/ruff#20374]
### Summary

Example of valid-and-intended format:

```python
class SimpleTestJob(EtlJob):
    """Test the `entry_target_runner.py` script and EtlJob class."""

    config_should_be_scheduled = False

    def extract(self):
        logger.info("This is the extract stage.")

    def transform(self):
        pass
```

Example of bad ordering:
```python
class SimpleTestJob(EtlJob):
    """Test the `entry_target_runner.py` script and EtlJob class."""

    def extract(self):
        logger.info("This is the extract stage.")

    config_should_be_scheduled = False

    def transform(self):
        pass
```

**Comment by MeGaGiGaGon** [comment]
There is a rule that checks for the ordering of class bodies, however it's django model specific: [django-unordered-body-content-in-model (DJ012)](https://docs.astral.sh/ruff/rules/django-unordered-body-content-in-model/#django-unordered-body-content-in-model-dj012)

**Comment by DeflateAwning** [comment]
Interesting case/example. Unfortunately does not help as it's extremely specific to Django, but I'm more interested in the general case for all classes.
